README.md

```diff
@@ -1,29 +1,75 @@
-jmx4prometheus
+JMX Exporter
 =====
 
-JMX to prometheus bridge.
+JMX to Prometheus bridge.
 
-A webserver that exposes mBeans of a JMX target in JSON format suitable
-for http://github.com/prometheus.
-
-# About
-A prometheus server makes a JSON request to a the jmx4prometheus server which then
-scrapes all beans of a target JMX port on another process. The target can
-either be statically configured in the configuration or passed in by
-prometheus server as a query string parameter.
-Jmx4prometheus can return all mBeans (that are numeric) or be configured with
-whitelist/blacklist behaviour.
+A Collector that can configurably scrape and expose mBeans of a JMX target.
+You can optionally run this as a HTTP server and/or scrape via remote JMX.
 
 ## Building and Running
 
 `mvn package` to build.
-See `run_jmx4prometheus_server.sh` for a sample script that runs the webserver.
+See `./run_sample_httpserver.sh` for a sample script that runs the httpserver against itself.
+
+## Configuration
+The configuration is in JSON, an example with all possible options:
+```
+{ 
+  "hostPort": "127.0.0.1:1234",
+  "rules": [
```

Comment:
This is very much reference documentation. I plan to write a blog post a few days after launch on how to use the JMX Exporter Java Agent with Cassandra that's more a tutorial/codelab, and ultimately move that to be part of the central Prometheus docs.